src/main/java/eu/europa/ted/eforms/sdk/component/SdkComponentFactory.java

```diff
@@ -126,22 +142,40 @@ protected <T> T getComponentImpl(String sdkVersion, final SdkComponentType compo
   }
 
   protected <T> T getComponentImpl(String sdkVersion, final SdkComponentType componentType,
-      final String qualifier, final Class<T> intf, Object... initArgs) throws InstantiationException {
+      final String qualifier, final Class<T> intf, Object... initArgs)
+      throws InstantiationException {
 
     String normalizedVersion = normalizeVersion(sdkVersion);
 
     ComponentSelector selector = new ComponentSelector(componentType, qualifier);
 
+    Map<ComponentSelector, SdkComponentDescriptor<?>> map =
+        Optional.ofNullable(componentsMap.get(normalizedVersion))
+            .orElseGet(Collections::emptyMap);
+
+    if (logger.isDebugEnabled()) {
+      logger.debug("selector componentType={}", selector.componentType);
+      logger.debug("selector qualifier={}", selector.qualifier);
+      logger.debug("normalized version={}", normalizedVersion);
```

Comment:
Those 3 lines should be merged into one, with a clearer text.
And for other logs, those values are in square brackets, so let's be consistent.
So for example:
"Looking for component with version=[{}], componentType=[{}], qualifier=[{}]"